doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -711,10 +711,66 @@ If `attrsFormat` is `values` then values entity partial representation mode is u
 Notifications must include a `X-Ngsiv2-AttrsFormat` header with the value of `attrsFormat` of the associated subscription,
 so notification receiver can know the format used without needing processing notification payload.
 
-### Notification templates
+## Custom notifications
 
-Based on the "template" field defined as part of the subscription and in a macro replacement language
-that could be used in "payload", "url", "headers", "method" and "qs". Details TBD.
+NGSIv2 clients can customize HTTP notification messages using a simple template mechanism. The
+`notification.httpCustom` field allows to specify the following fields subject to be templatized:
```

Comment:
notification.httpCustom`  property  of a subscription allows to specify allows to specify the following parameters